hack/ci-e2e-lib.sh

```diff
@@ -32,6 +32,9 @@ k8s::prepareKindestImagesVariables() {
     if [[ "${GINKGO_FOCUS:-}" == "\\[supervisor\\]" ]] || [[ "${GINKGO_FOCUS:-}" == "" ]]; then
       KUBERNETES_VERSION_LATEST_CI=$(grep KUBERNETES_VERSION_LATEST_CI: < "$E2E_CONF_FILE" | awk -F'"' '{ print $2}')
       echo "Defaulting KUBERNETES_VERSION_LATEST_CI to ${KUBERNETES_VERSION_LATEST_CI} to trigger image build (because env var is not set)"
+
+      KUBERNETES_VERSION_MANAGEMENT_LATEST_CI=$(grep KUBERNETES_VERSION_MANAGEMENT_LATEST_CI: < "$E2E_CONF_FILE" | awk -F'"' '{ print $2}')
```

Comment:
I assume in both of these cases we are running the clusterctl upgrade tests?

Reply:
Probably worth opening an issue to do the same refactoring as in core CAPI to make this easier

Reply:
Ah could it be that the first one doesn't need KUBERNETES_VERSION_LATEST_CI? (I think conformance is skipped there)